### ISSUE TITLE
Errors on Phoenix LiveView components

### ISSUE THREAD

**Issue by nelsonmestevao** [adobe/elixir-styler#36]
## Elixir / OTP Version

**Elixir**: `1.14.4`
**OTP**: `25.3.2`

## Example Input

New Phoenix 1.7.2 app.

## Current behaviour / Stacktrace

```elixir
** (SyntaxError) :1:1: syntax error before: '<'
    |
  1 | <.flash_group flash={@flash} />
    | ^
    lib/styler.ex:33: Styler.format/2
    lib/mix/tasks/style.ex:96: Mix.Tasks.Style.style_file/3
    (elixir 1.14.4) lib/task/supervised.ex:89: Task.Supervised.invoke_mfa/2
    (elixir 1.14.4) lib/task/supervised.ex:34: Task.Supervised.reply/4
    (stdlib 4.3.1) proc_lib.erl:240: :proc_lib.init_p_do_apply/3
```

## Expected Output

I think probably it is because I cannot use it with `HEEx` files, however I don't find any way to use styler only `.{ex,exs}` files.

**Comment by novaugust** [comment]
Thanks for the report -- this is just a `mix style` bug that prompted us to make styler a plugin, sorry for forgetting about it :) fixed in e9510eda14ee376705ae802d29cb779d2fc10f23

**Comment by nelsonmestevao** [comment]
Thank you 🙏
